sandbox/registration-processor-mz.properties

```diff
@@ -465,6 +465,10 @@ mosip.kernel.jsonvalidator.file-storage-uri=${spring.cloud.config.uri}/${spring.
 
 registration.processor.validateApplicantDocument=true
 
+# To enable/disable trust validation of a digital id signature
+mosip.regproc.packet-validator.signature.disable-trust-validation=false
+mosip.regproc.common.before-cbeff-others-attibute.reg-client-versions=1.2.0-SNAPSHOT
```

Comment:
This is common property for backward compatibility check, so add this before " WORKFLOW ENGINE PROPERTIES " section.
Change the value to 1.1.3,1.1.4,1.1.5
Please add appropriate comment that describes the property and format of the value to be given

Reply:
corrected as mentioned...

Reply:
Change the comments to "List of old reg client versions with others attribute not present but currently supported for registration, this is required to take care of the backward compatibility checks"

Reply:
changed the comment.